fabtests/benchmarks/benchmark_shared.c

```diff
@@ -239,6 +238,12 @@ int pingpong(void)
 		ret = pingpong_no_pre_posted_rx(inject_size);
 		if (ret)
 			return ret;
+
+		/* Final inband sync requires preposted RX buffers */
```

Comment:
It seems wrong that you are adding this in one benchmark `pingpong()`, and modifying its use in common code which gets called in every benchmark. `ft_finalize()` -> `ft_check_for_unexpected_comps()`

Reply:
There should be an outstanding rx ahead in `ft_enable_ep_recv`, do we really need this additional rx here?

Reply:
I have an impression we're consuming all RX-es at `pingpong_no_pre_posted_rx()` here:
https://github.com/ofiwg/libfabric/blob/99a50e7c2f9afd352ec3c74af66dfca35be23126/fabtests/benchmarks/benchmark_shared.c#L166-L167
https://github.com/ofiwg/libfabric/blob/99a50e7c2f9afd352ec3c74af66dfca35be23126/fabtests/benchmarks/benchmark_shared.c#L179-L180

I contrast to `pingpong_pre_posted_rx()` which reposts RX buffer in `ft_rx()`:
https://github.com/ofiwg/libfabric/blob/99a50e7c2f9afd352ec3c74af66dfca35be23126/fabtests/common/shared.c#L2588-L2610